README.md

```diff
@@ -1,11 +1,30 @@
 # Etcd
 
-> This charm was built from layers! To see the developer docs see `hacking.md` in
-the charm root.
+Etcd is a highly available distributed key value store that provides a reliable way to store data across a cluster of machines. Etcd gracefully handles master elections during network partitions and will tolerate machine failure, including the master.
+
+Your applications can read and write data into etcd. A simple use-case is to store database connection details or feature flags in etcd as key value pairs. These values can be watched, allowing your app to reconfigure itself when they change.
+
+Advanced uses take advantage of the consistency guarantees to implement database master elections or do distributed locking across a cluster of workers.
+
+Etcd allows storing data in a distributed hierarchical database with observation.
+
+# Usage
+
+We can deploy a single node easily with
+
+```shell
+juju deploy etcd
+```
+Add and capacity with:
+
+```shell
+juju add-unit -n 2 etcd
+```
+
+Its recommended to run an odd number of machines as it has greater redundancy than even number (ie. 4, you can lose 1 before quorum is lost, where as 5, you can 2).
```

Comment:
Should be It's
s/even/an even/
s/4/with 4/
s/where as 5, you can/whereas with 5, you can lose/